### ISSUE TITLE
dronedj.com

### ISSUE THREAD

**Issue by Yuki2718** [uBlockOrigin/uAssets#7124]
### URL(s) where the issue occurs

`https://dronedj.com/`
`https://ww.dronedj.com/`

### Describe the issue

Ads ~~and placeholder~~: they seem to use obfuscation/ad-proxying.

### Screenshot(s)

<details>
<summary>SS on Firefox</summary>

![dronedj](https://user-images.githubusercontent.com/58900598/77183548-d4618b80-6b11-11ea-9920-75423be8ade0.png)

</details>

<details>
<summary>SS on Brave (animated ads)</summary>

![dronedj2](https://user-images.githubusercontent.com/58900598/77185881-759e1100-6b15-11ea-9a6f-19a410d625d3.png)

</details>

<details>
<summary>SS on Brave (the same ads)</summary>

![dronedj3](https://user-images.githubusercontent.com/58900598/77185906-7fc00f80-6b15-11ea-9bc4-01752882469c.png)

</details>

For the ad in first SS, you need to click "EXPAND FULL STORY +" on. ~~For the 2nd and 3rd, scroll down enough~~. Ads may change over time or by browser.

EDIT 2: for some reason now I don't see the 2nd/3rd ad - IDK why.

### Versions

- Browser/version: Firefox 74.0/Brave 1.5.113/Firefox Android 68.6
- uBlock Origin version: 1.25.2 on Fx/ 1.24.4 on Brave

### Settings

- Reproduced by default settings

### Notes

~~I omitted SS for placeholders, but they're bit different on each browser.~~ It seems what I thought to be placeholders are actually not.

**Comment by NubN** [comment]
this site has other "domains"

example:

`https://ww.9to5google.com/2020/03/20/youtube-video-quality-europe-coronavirus/`

**Comment by uBlock-user** [comment]
`https://ww.dronedj.com/` is the redirected domain after anti-adblock detecttion, infested with ads and crap. Same for the rest of the 9to5group domains.

**Comment by uBlock-user** [comment]
They tried with inline-script but they were patched with uBO filters, so they're doing via 1st party scripts.

**Comment by Yuki2718** [comment]
Yeah, but I was redirected to ww. even when I accessed the site with uBO disabled. I simply couldn't figured out "the best filter" - some scripts such as those start from `dronedj.com/9:U3Hh` can be blocked, but blocking all those start from `dronedj.com/9:jQA0` or whole inline-script seems to cause some breakages. Also sometimes "#" is appended to URL, other times "#aprd", IDK.

**Comment by Yuki2718** [comment]
I leave it to you - it's over my head ; )

**Comment by uBlock-user** [comment]
Blocking all `_static` folder scripts with `||dronedj.com/_static/$script,1p` stops the redirection completely but breaks search, black/white theme, expand story button won't do the animation etc..

**Comment by Yuki2718** [comment]
> 
> 
> Blocking all `_static` folder scripts with `||dronedj.com/_static/$script,1p` stops the redirection completely but breaks search, black/white theme, expand story button won't do the animation etc..

I'm still redirected to ww. with `||dronedj.com/_static/`. Cache, cookie etc. cleared.

**Comment by uBlock-user** [comment]
Those ad banners are blocked on my end with `dronedj.com##a[target="_blank"][rel="noopener noreferrer"]`

**Comment by llacb47** [comment]
```
||9to5google.com^$inline-script
||dronedj.com^$inline-script
```

Terrible solution, but seems to work OK for now.

**Comment by Yuki2718** [comment]
Thx for the fix, now redirect doesn't happen - but some ads are still sitting there.

![dronedj](https://user-images.githubusercontent.com/58900598/77219225-1375f800-6b77-11ea-8e1c-fdc009904ba1.png)
 
BTW shouldn't the resource to be redirected for xhr be noop.txt than noop.js?

**Comment by mapx-** [comment]
Well, it's kind of review and that insertion is still about drones.

**Comment by mapx-** [comment]
if xhr has still a script content the redirection is like for scripts

Actually there are 19 occurrences of `xhr,redirect=noopjs` in uBO filters list

**Comment by Yuki2718** [comment]
@mapx- Thank you for clarification!
